### DISCUSSION TITLE
Reboot Recovery

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#29] [Fixed Issues]
After a reboot, or power down, the Snap Client feature is enabled, and the NAS does not remount.

**Comment by ceejayemm** [comment]
@R101-github / @rern

I see similar things on the Features page (see #26) but my NAS source (mounted via NFS IS retained.

Chris

**Reply by Sagnasty** [reply]
I am seeing the snap client, file sharing (smb) and Wi-FI being enabled after reboot / cycle power.

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Reply by R101G** [reply]
To clarify my settings, I have smb enabled, and NAS is CIFS. The NAS mounts OK from the sources menu. My onboard wifi is disabled in boot/config.txt.

**Comment by rern** [comment]
Fixed.
